### PR TITLE
Some Medchem Mix Messages

### PR DESCRIPTION
**Pull request by Kmsxkuse**
:cl:
add: Added more flavor text when medical chems are created.
/:cl:

### CODE REVIEW

File: code/modules/reagents/chemistry/recipes/medicine.dm [hunk 1]
```diff
@@ -5,50 +5,57 @@
...
 	results = list("rezadone" = 3)
 	required_reagents = list("carpotoxin" = 1, "cryptobiolin" = 1, "copper" = 1)
+	mix_message = "The mixture hardens into a fishy smelling powder."
 
 /datum/chemical_reaction/spaceacillin
 	name = "Spaceacillin"
 	id = "spaceacillin"
 	results = list("spaceacillin" = 2)
 	required_reagents = list("cryptobiolin" = 1, "epinephrine" = 1)
+	mix_message = "A light pink precipitate quickly froms from the mixing of the two reagents."
```
Comment (ike709) [inline code/modules/reagents/chemistry/recipes/medicine.dm]: >froms from

File: code/modules/reagents/chemistry/recipes/medicine.dm [hunk 2]
```diff
@@ -67,6 +74,7 @@
 	id = "synthflesh"
 	results = list("synthflesh" = 3)
 	required_reagents = list("blood" = 1, "carbon" = 1, "styptic_powder" = 1)
+	mix_message = "The mixture quickly solidifies into a skin coloured mass."
```
Comment (ike709) [inline code/modules/reagents/chemistry/recipes/medicine.dm]: >coloured

File: code/modules/reagents/chemistry/recipes/medicine.dm [hunk 3]
```diff
@@ -80,37 +88,43 @@
...
 	results = list("pen_acid" = 6)
 	required_reagents = list("welding_fuel" = 1, "chlorine" = 1, "ammonia" = 1, "formaldehyde" = 1, "sodium" = 1, "cyanide" = 1)
+	mix_message = "An increasing acidic smell wafts from the container."
 
 /datum/chemical_reaction/sal_acid
 	name = "Salicyclic Acid"
 	id = "sal_acid"
 	results = list("sal_acid" = 5)
 	required_reagents = list("sodium" = 1, "phenol" = 1, "carbon" = 1, "oxygen" = 1, "sacid" = 1)
+	mix_message = "The mixture forms a bubbling opaque grey solution."
```
Comment (ike709) [inline code/modules/reagents/chemistry/recipes/medicine.dm]: >grey

File: code/modules/reagents/chemistry/recipes/medicine.dm [hunk 4]
```diff
@@ -171,58 +188,68 @@
...
 	results = list("antitoxin" = 3)
 	required_reagents = list("nitrogen" = 1, "silicon" = 1, "potassium" = 1)
+	mix_message = "The mixture pops and a fizzles to life."
 
 /datum/chemical_reaction/tricordrazine
 	name = "Tricordrazine"
 	id = "tricordrazine"
 	results = list("tricordrazine" = 3)
 	required_reagents = list("bicaridine" = 1, "kelotane" = 1, "antitoxin" = 1)
+	mix_message = "The mixture passes through various colours of the rainbow before settling on a light pink."
```
Comment (ike709) [inline code/modules/reagents/chemistry/recipes/medicine.dm]: >colours

File: code/modules/reagents/chemistry/recipes/medicine.dm [hunk 5]
```diff
@@ -171,58 +188,68 @@
...
 
 /datum/chemical_reaction/corazone
 	name = "Corazone"
 	id = "corazone"
 	results = list("corazone" = 3)
 	required_reagents = list("phenol" = 2, "lithium" = 1)
+	mix_message = "A distinctly white powder float to the top of the mixture."
```
Comment (ike709) [inline code/modules/reagents/chemistry/recipes/medicine.dm]: Should be floats

**Comment by ike709** [comment]
80% of my comments are just pointing things out that affront my freedom but there are some actual issues.

**Comment by Kmsxkuse** [comment]
Derp derp, my english is failing me. Time to fix it now that it's a new day.

File: code/modules/reagents/chemistry/recipes/medicine.dm (hunk 2 above)
Comment (Kmsxkuse) [inline code/modules/reagents/chemistry/recipes/medicine.dm]: Oh come on.

File: code/modules/reagents/chemistry/recipes/medicine.dm (hunk 3 above)
Comment (Kmsxkuse) [inline code/modules/reagents/chemistry/recipes/medicine.dm]: What the heck is "gray"? Isn't grey the proper spelling throughout the world?

File: code/modules/reagents/chemistry/recipes/medicine.dm [hunk 6]
```diff
@@ -80,37 +88,43 @@
...
 	results = list("potass_iodide" = 2)
 	required_reagents = list("potassium" = 1, "iodine" = 1)
+	mix_message = "A bright green fluid collects at the bottom of the container."
 
 /datum/chemical_reaction/pen_acid
 	name = "Pentetic Acid"
 	id = "pen_acid"
 	results = list("pen_acid" = 6)
 	required_reagents = list("welding_fuel" = 1, "chlorine" = 1, "ammonia" = 1, "formaldehyde" = 1, "sodium" = 1, "cyanide" = 1)
+	mix_message = "An increasing acidic smell wafts from the container."
```
Comment (AuroranAI) [inline code/modules/reagents/chemistry/recipes/medicine.dm]: Increasingly

**Pull request merged**